test/models/subscription_status_test.rb

```diff
@@ -17,27 +17,32 @@ def test_get_status
     end
 
     def test_to_status_valid
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_VALID)
       assert_equal Katello::SubscriptionStatus::VALID, status.to_status
     end
 
     def test_to_status_disabled
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_DISABLED)
       assert_equal Katello::SubscriptionStatus::DISABLED, status.to_status
     end
 
     def test_to_status_partial
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_PARTIAL)
       assert_equal Katello::SubscriptionStatus::PARTIAL, status.to_status
     end
 
     def test_to_status_invalid
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_INVALID)
       assert_equal Katello::SubscriptionStatus::INVALID, status.to_status
     end
 
     def test_to_status_unsubscribed_hypervisor
-      stub_status('unsubscribed_hypervisor')
+      host.organization.stubs(:simple_content_access?).returns(false)
+      host.subscription_facet.stubs(:unsubscribed_hypervisor?).returns(true)
```

Comment:
Are both stubs needed?

Reply:
I think you had a new test case here - where'd it go? 🔍

Reply:
Got lost with my editing, adding it back and removed the uneeded sub on line 44